Last login: Fri Apr 28 20:27:56 on ttys002
deanzhangdeMacBook-Pro:~ deanzhang$ cd Desktop/1111/
deanzhangdeMacBook-Pro:1111 deanzhang$ python trainedmodel.py 
Using TensorFlow backend.
Traceback (most recent call last):
  File "trainedmodel.py", line 21, in <module>
    import imdbReview
ImportError: No module named 'imdbReview'
deanzhangdeMacBook-Pro:1111 deanzhang$ python trainedmodel.py 
Using TensorFlow backend.
/Users/deanzhang/anaconda/lib/python3.5/site-packages/keras/preprocessing/text.py:89: UserWarning: The `nb_words` argument in `Tokenizer` has been renamed `num_words`.
  warnings.warn('The `nb_words` argument in `Tokenizer` '
Found 123417 unique tokens.
y_train shape: (100000, 5)
y_test shape: (100000, 5)
Traceback (most recent call last):
  File "trainedmodel.py", line 100, in <module>
    f = open('./glove.6B.100d.txt',encoding='utf8')
FileNotFoundError: [Errno 2] No such file or directory: './glove.6B.100d.txt'
deanzhangdeMacBook-Pro:1111 deanzhang$ python trainedmodel.py 
Using TensorFlow backend.
/Users/deanzhang/anaconda/lib/python3.5/site-packages/keras/preprocessing/text.py:89: UserWarning: The `nb_words` argument in `Tokenizer` has been renamed `num_words`.
  warnings.warn('The `nb_words` argument in `Tokenizer` '
Found 123417 unique tokens.
y_train shape: (100000, 5)
y_test shape: (100000, 5)
Traceback (most recent call last):
  File "trainedmodel.py", line 100, in <module>
    f = open('./glove.6B.100d.txt',encoding='utf8')
FileNotFoundError: [Errno 2] No such file or directory: './glove.6B.100d.txt'
deanzhangdeMacBook-Pro:1111 deanzhang$ python trainedmodel.py 
Using TensorFlow backend.
/Users/deanzhang/anaconda/lib/python3.5/site-packages/keras/preprocessing/text.py:89: UserWarning: The `nb_words` argument in `Tokenizer` has been renamed `num_words`.
  warnings.warn('The `nb_words` argument in `Tokenizer` '
Found 123417 unique tokens.
y_train shape: (100000, 5)
y_test shape: (100000, 5)
Found 400000 word vectors.
Traceback (most recent call last):
  File "trainedmodel.py", line 114, in <module>
    embedding_matrix[i] = embedding_vector
ValueError: could not broadcast input array from shape (300) into shape (100)
deanzhangdeMacBook-Pro:1111 deanzhang$ python trainedmodel.py 
Using TensorFlow backend.
/Users/deanzhang/anaconda/lib/python3.5/site-packages/keras/preprocessing/text.py:89: UserWarning: The `nb_words` argument in `Tokenizer` has been renamed `num_words`.
  warnings.warn('The `nb_words` argument in `Tokenizer` '
Found 123417 unique tokens.
y_train shape: (100000, 5)
y_test shape: (100000, 5)
Found 400000 word vectors.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
_________________________________________________________________
Layer (type)                 Output Shape              Param #   
=================================================================
embedding_1 (Embedding)      (None, None, 300)         37025400  
_________________________________________________________________
conv1d_1 (Conv1D)            (None, None, 100)         90100     
_________________________________________________________________
max_pooling1d_1 (MaxPooling1 (None, None, 100)         0         
_________________________________________________________________
dropout_1 (Dropout)          (None, None, 100)         0         
_________________________________________________________________
lstm_1 (LSTM)                (None, 100)               80400     
_________________________________________________________________
dense_1 (Dense)              (None, 5)                 505       
=================================================================
Total params: 37,196,405
Trainable params: 171,005
Non-trainable params: 37,025,400
_________________________________________________________________
Train...
/Users/deanzhang/anaconda/lib/python3.5/site-packages/keras/models.py:834: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  warnings.warn('The `nb_epoch` argument in `fit` '
Epoch 1/3
  1504/100000 [..............................] - ETA: 1512s - loss: 1.4031 - acc: 0.4109^CTraceback (most recent call last):
  File "trainedmodel.py", line 142, in <module>
    nb_epoch=3
  File "/Users/deanzhang/anaconda/lib/python3.5/site-packages/keras/models.py", line 853, in fit
    initial_epoch=initial_epoch)
  File "/Users/deanzhang/anaconda/lib/python3.5/site-packages/keras/engine/training.py", line 1486, in fit
    initial_epoch=initial_epoch)
  File "/Users/deanzhang/anaconda/lib/python3.5/site-packages/keras/engine/training.py", line 1141, in _fit_loop
    outs = f(ins_batch)
  File "/Users/deanzhang/anaconda/lib/python3.5/site-packages/keras/backend/tensorflow_backend.py", line 2103, in __call__
    feed_dict=feed_dict)
  File "/Users/deanzhang/anaconda/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 767, in run
    run_metadata_ptr)
  File "/Users/deanzhang/anaconda/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 965, in _run
    feed_dict_string, options, run_metadata)
  File "/Users/deanzhang/anaconda/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 1015, in _do_run
    target_list, options, run_metadata)
  File "/Users/deanzhang/anaconda/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 1022, in _do_call
    return fn(*args)
  File "/Users/deanzhang/anaconda/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 1004, in _run_fn
    status, run_metadata)
KeyboardInterrupt
Exception ignored in: <bound method BaseSession.__del__ of <tensorflow.python.client.session.Session object at 0x15f4d8e10>>
Traceback (most recent call last):
  File "/Users/deanzhang/anaconda/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 582, in __del__
AttributeError: 'NoneType' object has no attribute 'TF_DeleteStatus'
deanzhangdeMacBook-Pro:1111 deanzhang$ python trainedmodel.py 
Using TensorFlow backend.
/Users/deanzhang/anaconda/lib/python3.5/site-packages/keras/preprocessing/text.py:89: UserWarning: The `nb_words` argument in `Tokenizer` has been renamed `num_words`.
  warnings.warn('The `nb_words` argument in `Tokenizer` '
Found 123417 unique tokens.
y_train shape: (100000, 5)
y_test shape: (100000, 5)
^CTraceback (most recent call last):
  File "trainedmodel.py", line 102, in <module>
    values = line.split()
KeyboardInterrupt
deanzhangdeMacBook-Pro:1111 deanzhang$ python trainedmodel.py 
Using TensorFlow backend.
/Users/deanzhang/anaconda/lib/python3.5/site-packages/keras/preprocessing/text.py:89: UserWarning: The `nb_words` argument in `Tokenizer` has been renamed `num_words`.
  warnings.warn('The `nb_words` argument in `Tokenizer` '
Found 123417 unique tokens.
y_train shape: (100000, 5)
y_test shape: (100000, 5)
Found 400000 word vectors.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
_________________________________________________________________
Layer (type)                 Output Shape              Param #   
=================================================================
embedding_1 (Embedding)      (None, None, 300)         37025400  
_________________________________________________________________
conv1d_1 (Conv1D)            (None, None, 100)         90100     
_________________________________________________________________
max_pooling1d_1 (MaxPooling1 (None, None, 100)         0         
_________________________________________________________________
dropout_1 (Dropout)          (None, None, 100)         0         
_________________________________________________________________
lstm_1 (LSTM)                (None, 100)               80400     
_________________________________________________________________
dense_1 (Dense)              (None, 5)                 505       
=================================================================
Total params: 37,196,405
Trainable params: 171,005
Non-trainable params: 37,025,400
_________________________________________________________________
Train...
Traceback (most recent call last):
  File "trainedmodel.py", line 143, in <module>
    validation_data=[X_test, y_test]
NameError: name 'X_test' is not defined
Exception ignored in: <bound method BaseSession.__del__ of <tensorflow.python.client.session.Session object at 0x15f319dd8>>
Traceback (most recent call last):
  File "/Users/deanzhang/anaconda/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 582, in __del__
AttributeError: 'NoneType' object has no attribute 'TF_DeleteStatus'
deanzhangdeMacBook-Pro:1111 deanzhang$ python trainedmodel.py 
Using TensorFlow backend.
/Users/deanzhang/anaconda/lib/python3.5/site-packages/keras/preprocessing/text.py:89: UserWarning: The `nb_words` argument in `Tokenizer` has been renamed `num_words`.
  warnings.warn('The `nb_words` argument in `Tokenizer` '
Found 123417 unique tokens.
y_train shape: (100000, 5)
y_test shape: (100000, 5)
Found 400000 word vectors.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
_________________________________________________________________
Layer (type)                 Output Shape              Param #   
=================================================================
embedding_1 (Embedding)      (None, None, 300)         37025400  
_________________________________________________________________
conv1d_1 (Conv1D)            (None, None, 100)         90100     
_________________________________________________________________
max_pooling1d_1 (MaxPooling1 (None, None, 100)         0         
_________________________________________________________________
dropout_1 (Dropout)          (None, None, 100)         0         
_________________________________________________________________
lstm_1 (LSTM)                (None, 100)               80400     
_________________________________________________________________
dense_1 (Dense)              (None, 5)                 505       
=================================================================
Total params: 37,196,405
Trainable params: 171,005
Non-trainable params: 37,025,400
_________________________________________________________________
Train...
/Users/deanzhang/anaconda/lib/python3.5/site-packages/keras/models.py:834: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  warnings.warn('The `nb_epoch` argument in `fit` '
Epoch 1/15
100000/100000 [==============================] - 1319s - loss: 0.9530 - acc: 0.5894     
Epoch 2/15
100000/100000 [==============================] - 1337s - loss: 0.8308 - acc: 0.6450     
Epoch 3/15
100000/100000 [==============================] - 1356s - loss: 0.7845 - acc: 0.6660     
Epoch 4/15
100000/100000 [==============================] - 1331s - loss: 0.7499 - acc: 0.6824     
Epoch 5/15
100000/100000 [==============================] - 1339s - loss: 0.7231 - acc: 0.6944    Epoch 6/15
100000/100000 [==============================] - 1349s - loss: 0.6970 - acc: 0.7047    Epoch 7/15
100000/100000 [==============================] - 1308s - loss: 0.6762 - acc: 0.7139    Epoch 8/15
100000/100000 [==============================] - 1260s - loss: 0.6577 - acc: 0.7233    Epoch 9/15
100000/100000 [==============================] - 1259s - loss: 0.6398 - acc: 0.7308    Epoch 10/15
100000/100000 [==============================] - 1279s - loss: 0.6223 - acc: 0.7389    Epoch 11/15
100000/100000 [==============================] - 1318s - loss: 0.6078 - acc: 0.7456    Epoch 12/15
100000/100000 [==============================] - 1311s - loss: 0.5946 - acc: 0.7502    Epoch 13/15
100000/100000 [==============================] - 1333s - loss: 0.5845 - acc: 0.7558    Epoch 14/15
100000/100000 [==============================] - 1310s - loss: 0.5728 - acc: 0.7597    Epoch 15/15
100000/100000 [==============================] - 1262s - loss: 0.5613 - acc: 0.7643    Saved model to disk
deanzhangdeMacBook-Pro:1111 deanzhang$ python trainedmodel.py 
Using TensorFlow backend.
/Users/deanzhang/anaconda/lib/python3.5/site-packages/keras/preprocessing/text.py:89: UserWarning: The `nb_words` argument in `Tokenizer` has been renamed `num_words`.
  warnings.warn('The `nb_words` argument in `Tokenizer` '
Found 123417 unique tokens.
y_train shape: (100000, 5)
y_test shape: (100000, 5)
Found 400000 word vectors.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
_________________________________________________________________
Layer (type)                 Output Shape              Param #   
=================================================================
embedding_1 (Embedding)      (None, None, 300)         37025400  
_________________________________________________________________
conv1d_1 (Conv1D)            (None, None, 100)         90100     
_________________________________________________________________
max_pooling1d_1 (MaxPooling1 (None, None, 100)         0         
_________________________________________________________________
dropout_1 (Dropout)          (None, None, 100)         0         
_________________________________________________________________
lstm_1 (LSTM)                (None, 100)               80400     
_________________________________________________________________
dense_1 (Dense)              (None, 5)                 505       
=================================================================
Total params: 37,196,405
Trainable params: 171,005
Non-trainable params: 37,025,400
_________________________________________________________________
Train...
/Users/deanzhang/anaconda/lib/python3.5/site-packages/keras/models.py:834: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.
  warnings.warn('The `nb_epoch` argument in `fit` '
Epoch 1/20
100000/100000 [==============================] - 1269s - loss: 0.9550 - acc: 0.5891    Epoch 2/20
100000/100000 [==============================] - 1263s - loss: 0.8291 - acc: 0.6471    Epoch 3/20
100000/100000 [==============================] - 1262s - loss: 0.7829 - acc: 0.6671    Epoch 4/20
100000/100000 [==============================] - 1262s - loss: 0.7491 - acc: 0.6810    Epoch 5/20
100000/100000 [==============================] - 1261s - loss: 0.7245 - acc: 0.6939    Epoch 6/20
100000/100000 [==============================] - 1261s - loss: 0.6996 - acc: 0.7062    Epoch 7/20
100000/100000 [==============================] - 1261s - loss: 0.6783 - acc: 0.7138    Epoch 8/20
100000/100000 [==============================] - 1262s - loss: 0.6573 - acc: 0.7248    Epoch 9/20
100000/100000 [==============================] - 1262s - loss: 0.6413 - acc: 0.7305    Epoch 10/20
100000/100000 [==============================] - 1262s - loss: 0.6209 - acc: 0.7384    Epoch 11/20
100000/100000 [==============================] - 1262s - loss: 0.6101 - acc: 0.7448    Epoch 12/20
100000/100000 [==============================] - 1262s - loss: 0.5972 - acc: 0.7497    Epoch 13/20
100000/100000 [==============================] - 1261s - loss: 0.5836 - acc: 0.7565    Epoch 14/20
100000/100000 [==============================] - 1262s - loss: 0.5711 - acc: 0.7625    Epoch 15/20
100000/100000 [==============================] - 1262s - loss: 0.5634 - acc: 0.7657    Epoch 16/20
100000/100000 [==============================] - 1260s - loss: 0.5520 - acc: 0.7717    Epoch 17/20
100000/100000 [==============================] - 1261s - loss: 0.5461 - acc: 0.7740    Epoch 18/20
100000/100000 [==============================] - 1261s - loss: 0.5380 - acc: 0.7772    Epoch 19/20
100000/100000 [==============================] - 1264s - loss: 0.5289 - acc: 0.7811    Epoch 20/20
100000/100000 [==============================] - 1266s - loss: 0.5218 - acc: 0.7848    Saved model to disk
Exception ignored in: <bound method BaseSession.__del__ of <tensorflow.python.client.session.Session object at 0x15ecd07f0>>
Traceback (most recent call last):
  File "/Users/deanzhang/anaconda/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 582, in __del__
UnboundLocalError: local variable 'status' referenced before assignment
deanzhangdeMacBook-Pro:1111 deanzhang$ cd test
-bash: cd: test: No such file or directory
deanzhangdeMacBook-Pro:1111 deanzhang$ cd testpy/
deanzhangdeMacBook-Pro:testpy deanzhang$ python excutable.py 
Using TensorFlow backend.
/Users/deanzhang/anaconda/lib/python3.5/site-packages/keras/preprocessing/text.py:89: UserWarning: The `nb_words` argument in `Tokenizer` has been renamed `num_words`.
  warnings.warn('The `nb_words` argument in `Tokenizer` '
Found 7 unique tokens.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
Loaded model from disk
1/1 [==============================] - 0s
[1]
Test accuracy: 0.0
Found 31 unique tokens.
Loaded model from disk
2/2 [==============================] - 0s
[1 1]
Test accuracy: 0.0
Found 31 unique tokens.
Loaded model from disk
2/2 [==============================] - 0s
[1 1]
Test accuracy: 0.0
Found 31 unique tokens.
Loaded model from disk
2/2 [==============================] - 0s
[1 1]
Test accuracy: 0.0
Found 31 unique tokens.
Loaded model from disk
2/2 [==============================] - 0s
[1 1]
Test accuracy: 0.0
Found 31 unique tokens.
Loaded model from disk
2/2 [==============================] - 0s
[1 1]
Test accuracy: 0.0
^C
^C
Traceback (most recent call last):
  File "excutable.py", line 116, in <module>
    if __name__ == '__main__':
  File "excutable.py", line 113, in main
    font.families()
  File "/Users/deanzhang/anaconda/lib/python3.5/tkinter/__init__.py", line 1131, in mainloop
    self.tk.mainloop(n)
KeyboardInterrupt
deanzhangdeMacBook-Pro:testpy deanzhang$ 
deanzhangdeMacBook-Pro:testpy deanzhang$ 
